### PR TITLE
Make SVG do the Work

### PR DESCRIPTION
**Pull request by coreyja**
Basically we make a `polyline` between all the centers of the squares that make up a snake
Then we set a stroke and no fill on the line
Finally we can set the linecap to complete the head and tail!

Example: 
image

### CODE REVIEW

**Comment by coreyja** [comment]
Now with 100% more roundness!

image

File: src/lib/components/Gameboard.svelte [hunk 1]
```diff
@@ -1,5 +1,23 @@
-<script>
-	export let frame;
+<script lang="ts">
```
Comment (coreyja) [inline src/lib/components/Gameboard.svelte:1]: My editor wasn't happy that this wasn't typed, so learned how to use TS in these components

No need to but as we've said before I like types 🤣

File: src/lib/components/Gameboard.svelte [hunk 2]
```diff
@@ -14,41 +32,55 @@
 	$: svgWidth = CELL_SIZE * frame.width + CELL_SPACING * frame.width + CELL_SPACING;
 	$: svgHeight = CELL_SIZE * frame.height + CELL_SPACING * frame.height + CELL_SPACING;
 
-	function svgCirclePropsAtCoords(x, y) {
-		const rect = svgRectPropsAtCoords(x, y);
+	function svgCirclePropsAtPoint({ x, y }: Point) {
```
Comment (coreyja) [inline src/lib/components/Gameboard.svelte:35]: I like passing in the whole object and then de-structuring it into the `x` and `y`!

This way we can pass in a Point without having to break it into the x and y

**Comment by bvanvugt** [comment]
Okay, these changes are in main now 😁  I moved the type definitions to the game store too.

**Pull request closed without merging**